### PR TITLE
feat(core): Support video task.

### PR DESCRIPTION
**Pull request by IRONICBo**
- Store ASR task to mem cache.
- Check ASR task status in daemon timed task.(Will update to callback api.)

### CODE REVIEW

File: internal/core/media.go [hunk 1]
```diff
@@ -183,32 +186,48 @@ func UploadFile(ctx *yap.Context, community *Community) {
...
 		xLog.Error("save file", err.Error())
 		ctx.JSON(500, err.Error())
 		return
 	}
+
+	// Judge the file type and start the corresponding task
+	fileType := http.DetectContentType(bytes)
+	if strings.Contains(fileType, "video") {
+		// Start captioning task
+		err := c.NewVideoTask(context.Background(), uid, strconv.Itoa(int(id)))
```
Comment (CarlJi) [inline internal/core/media.go]: recommend to use `strconv.FormatInt` to avoid data corruption

File: internal/core/task.go [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+	defer ticker.Stop()
+	timer := time.NewTimer(timeout)
+	defer timer.Stop()
+
+	for {
+		select {
+		case <-ticker.C:
+			c.xLog.Infof("TimedCheckVideoTask start, timeout: %v", timeout)
+			// Iterator video task cache
+			c.translation.VideoTaskCache.RLock()
```
Comment (CarlJi) [inline internal/core/task.go]: this lock seems too big

Comment (IRONICBo) [reply internal/core/task.go]: This issue has been solved.
image

**Review by CarlJi — APPROVED** [review]
LGTM

**Pull request merged**